.github/workflows/bb_containers.yml

```diff
@@ -103,7 +103,9 @@ jobs:
             exit 1
           }
           TAG_TMP=${{ matrix.image }}
-          if [[ $TAG_TMP = "ubi8" ]]; then
+          if [[ $TAG_TMP = "quay.io/centos/centos:stream8" ]]; then
+            echo "TAG=centos8" >>$GITHUB_ENV
```

Comment:
Maybe ?
```diff
- echo "TAG=centos8" >>$GITHUB_ENV
+ echo "TAG=centos8-stream" >>$GITHUB_ENV
```

This way we do not overwrite actual centos8 image on quay.io and we know better what we are talking about.

Reply:
I think there's no question that centos8 is eol and will be remove from bb-workers.

For the moment I was trying to get https://buildbot.mariadb.org/#/workers/538 green again based on a hopefully fixed gcc version in centos stream8 (because RHEL has had it for a while).
https://jira.mariadb.org/browse/MDEV-27008

If we continue with centos8-stream and the associated bb changes, sure centos8-stream is a better name.